SchedulerJobs/SchedulerJobs.Services/ReconcileHearingAudioService.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using VideoApi.Client;
+
+namespace SchedulerJobs.Services
+{
+
+    public interface IReconcileHearingAudioService
```

Comment:
Could you please extract interface to separate file?

Reply:
fixed